client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -184,6 +184,13 @@
               target="_blank">
               {{ Translator.trans('original.pdf') }}
             </a>
+            <a
+              v-if="hasPermission('area_admin_original_statement_list')"
```

Comment:
I think it cannot be a mistake to add `data-cy="listStatements:originalStatement"` here.

Reply:
Thanks, I forgot about that. Done in [0ef94b2](https://github.com/demos-europe/demosplan-core/pull/3884/commits/0ef94b2ea5de29465afb36b37d086f4d509b0384)